lib/hacluster.pm

```diff
@@ -311,7 +311,7 @@ sub get_lun {
 sub pre_run_hook {
     my ($self) = @_;
 
-    $prev_console = $testapi::selected_console;
+    $prev_console = $autotest::selected_console;
```

Comment:
is this substitution really related to perltidy?

Reply:
@Soulofdestiny it's related to https://github.com/os-autoinst/os-autoinst/pull/985#discussion_r203073837.

Reply:
I recommend to use: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/5412/files instead :)

Reply:
Yes should be track in another PR, and also SLES4SAP code needs to be change as well.

I will revert this part, as Qemu changes are not merged yet on o.s.d.